### PR TITLE
Sketches

### PR DESCRIPTION
**Pull request by wissog**
Added multiple language support for images

### CODE REVIEW

File: multiple_zip_query.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+SELECT
+  cu.id,
```
Comment (edoshor) [inline multiple_zip_query.sql]: Please remove this file as it has nothing to do in SCM

File: src/components/shared/UnitMaterials/Sketches.js [hunk 2]
```diff
@@ -4,14 +4,15 @@ import { bindActionCreators } from 'redux';
...
 import 'react-image-gallery/styles/css/image-gallery.css';
-
 import { RTL_LANGUAGES } from '../../../helpers/consts';
 import { formatError } from '../../../helpers/utils';
 import { assetUrl, imaginaryUrl, Requests } from '../../../helpers/Api';
 import { actions, selectors } from '../../../redux/modules/assets';
 import { selectors as settings } from '../../../redux/modules/settings';
 import * as shapes from '../../shapes';
 import { ErrorSplash, FrownSplash, LoadingSplash } from '../../shared/Splash';
+import { Container, Divider, Segment } from 'semantic-ui-react';
```
Comment (edoshor) [inline src/components/shared/UnitMaterials/Sketches.js]: This one belongs to the top import section - 3rd parties

File: src/components/shared/UnitMaterials/Sketches.js [hunk 3]
```diff
@@ -118,33 +127,47 @@ class Sketches extends React.Component {
       const items = imageObjs
         .map(imageGalleryItem)
         .sort((a, b) => {
-          if (a.original < b.original) {
+          if (a.original < b.original)
```
Comment (edoshor) [inline src/components/shared/UnitMaterials/Sketches.js]: Please keep the curly braces `{` in place

**Comment by wissog** [comment]
i'm done with all this.
give me new task

*Gabi*

*"What R we living for?" :)*

**Pull request merged**